android/app/build.gradle

```diff
@@ -252,6 +252,12 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        // We need a custom build type, so we can allow http clear text traffic in a release build:
```

Comment:
Do we still need it?

Reply:
oh good catch, let me double check! Might be still needed as we call to our localhost HTTP server

Reply:
Yes, this is still needed